src/transport/PairingSession.h

```diff
@@ -97,7 +104,7 @@ class DLL_EXPORT PairingSession
     void SetPeerNodeId(NodeId peerNodeId) { mPeerNodeId = peerNodeId; }
     void SetPeerCATs(CATValues peerCATs) { mPeerCATs = peerCATs; }
     void SetPeerSessionId(uint16_t id) { mPeerSessionId.SetValue(id); }
-    void SetLocalSessionId(uint16_t id) { mLocalSessionId = id; }
+    void SetSecureSessionHolder(SessionHolder & holder) { mSecureSessionHolder = holder; }
```

Comment:
Again, should this be taking an rvalue and named something like `TakeSecureSession` or something?